tests/unit/components/AdjustableSidebarWidth.spec.js

```diff
@@ -382,4 +382,28 @@ describe('AdjustableSidebarWidth', () => {
     expect(FocusTrap.mock.results[0].value.stop).toHaveBeenCalledTimes(1);
     expect(changeElementVOVisibility.show).toHaveBeenCalledTimes(1);
   });
+
+  it('accounts for zoomed in devices', () => {
+    window.scrollX = 55;
+    const wrapper = createWrapper();
+    const aside = wrapper.find('.aside');
+    // assert dragging
+    wrapper.find('.resize-handle').trigger('touchstart', { type: 'touchstart' });
+    document.dispatchEvent(createEvent(eventsMap.touch.move, {
+      touches: [{
+        clientX: 300,
+      }],
+    }));
+    // assert class
+    expect(aside.classes()).toContain('dragging');
+    // offset is 100, so we remove it from the clientX, but we add the scrollX.
```

Comment:
Could you explain what the offset is referring to here? Is it the `sidebar.offsetLeft`?

Reply:
the `element.offsetLeft`